### PR TITLE
Add publishing for 7.0.3xx + 8.0 P1-4

### PR DESCRIPTION
**Pull request by mmitche**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation

### CODE REVIEW

**Comment by mmitche** [comment]
Resolves https://github.com/dotnet/arcade/issues/12184

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs [hunk 1]
```diff
@@ -616,92 +616,112 @@ public enum BuildQuality
...
 
-            // .NET 7.0.1xx SDK,
+            // .NET 7.0.2xx SDK,
             new TargetChannelConfig(
-                id: 2237,
+                id: 3259,
                 isInternal: false,
                 publishingInfraVersion: PublishingInfraVersion.All,
-                akaMSChannelNames: new List<string>() { "7.0.1xx", "7.0" },
+                akaMSChannelNames: new List<string>() { "7.0.2xx" },
```
Comment (riarenas) [inline src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs:644]: No 7.0 aka.ms channel for this one?

Comment (riarenas) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs:644]: or for the internal version?

Comment (mmitche) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs:644]: 7.0 isn't RTM yet. At that point, it will get the 7.0 tag.

Comment (riarenas) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs:644]: LGTM then!

**Pull request merged**